app/views/posts/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="container content">
+  <div class="row">
+    <div class="col-sm-8 content">
+      <main>
+        <h1><%= @current_post.title.upcase %></h1>
+        <div class="subheading">
+          <%= image_tag(@current_post.author.parameterize, class: 'avatar') %>
+          <p><%= @current_post.author %> <span>on <%= @current_post.created_at.strftime("%A %b %d %Y %-I:%M %p") %></span></p>
+        </div>
+        <%= simple_format @current_post.first_half(@current_post.body) %>
+        <div class="interject">
```

Comment:
What if a post has no interject?

Or, are you expecting that a post has to have an interject?